### PR TITLE
Having buildozer ignore whl file extensions to prevent bloated apks

### LINKED ISSUE #17: whl file is bundled into apk
pasted image at 2017_10_12 07_45 am

### PR DESCRIPTION
**Pull request by DXCanas**
Resolves #17

### CODE REVIEW

**Comment by DXCanas** [comment]
also TODO: shrink further by removing some of the crypt stuff

File: Makefile [hunk 1]
```diff
@@ -18,7 +18,7 @@ replaceloadingpage:
 
 # Extract the whl file
 extractkolibriwhl:
-	unzip -q "src/kolibri*.whl" "kolibri/*" -d src/
+	unzip -q "src/kolibri*.whl" "kolibri/*" -x /kolibri/dist/cext/ -d src/
```
Comment (christianmemije) [inline Makefile]: Did you test this?

Comment (christianmemije) [reply Makefile]:
```

Comment (christianmemije) [reply Makefile]:
```

Comment (christianmemije) [reply Makefile]:
```

Comment (christianmemije) [reply Makefile]:
```
lucye:/data/data/org.le.kolibri/files/kolibri/dist $ ls                        
CherryPy-6.2.0.post20171012-py2.7.egg-info le_utils                     
CherryPy-6.2.0.post20171016-py2.7.egg-info metaphone                    
CherryPy-6.2.0.post20171017-py2.7.egg-info morango                      
README                                     morango-0.0.2-py2.7.egg-info 
__init__.py                                mptt                         
args.py                                    porter2stemmer               
cext                                       pyasn1                       
cherrypy                                   pytz                         
clint                                      requests                     
colorlog                                   requests_toolbelt            
concurrent                                 rest_framework               
dateutil                                   rest_framework_csv           
django                                     rsa                          
django_filters                             six.py                       
django_js_reverse                          sqlalchemy                   
docopt.py                                  tests                        
iceqube                                    tqdm                         
ifcfg                                      tzlocal                      
jsonfield                                  unicodecsv                   
kolibri_exercise_perseus_plugin         
```

Comment (christianmemije) [reply Makefile]: `unzip -q "src/kolibri*.whl" "kolibri/*" -x */cext/* -d src/`

Comment (DXCanas) [reply Makefile]: I did test this, and I didn't see it unzipped o.O 

Excluding a directory should be automatically recursive, per the `unzip` docs. I'll look into it

**Comment by DXCanas** [comment]
Turns out I had some local changes that were outside the commit :confused: 

Not a fan of using something like `*cext*`; could have unintended consequences

**Comment by christianmemije** [comment]
Shrinks apk size from 156 mb to 36 mb :1st_place_medal:

**Pull request merged**